MAUI/Maps/tooltip.md

```diff
@@ -0,0 +1,468 @@
+               new Model("North America", 472,"8,600,000 sq. km."),
+               new Model("South America", 363,"10,180,000 sq. km."),
+               new Model("Australia", 348,"59,180,000 sq. km."),
+          };
+
+        SfMaps maps = new SfMaps();
+        MapShapeLayer layer = new MapShapeLayer();
+        layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+        layer.PrimaryValuePath = "Continent";
+        layer.ShapeDataField = "continent";
+        layer.DataSource = Data;
+        layer.ShowShapeTooltip = true;
+        layer.ShapeTooltipTemplate = CreateDataTemplate();
+
+        maps.Layer = layer;
+        this.Content = maps;
+    }
+    private DataTemplate CreateDataTemplate()
+    {
+        return new DataTemplate(() =>
+        {
+            var grid = new Grid()
+            {
+                RowDefinitions =
+                {
+                  new RowDefinition { Height = GridLength.Auto },
+                  new RowDefinition{ Height = GridLength.Auto },
+                  new RowDefinition { Height = GridLength.Auto }
+                },
+                ColumnDefinitions =
+                {
+                    new ColumnDefinition{ Width = 10},
+                    new ColumnDefinition{ Width = GridLength.Auto},
+                }
+            };
+            grid.WidthRequest = 120;
+            var image = new Image { Source = "flag.png", WidthRequest = 20, HeightRequest = 20 };
+            grid.SetRow(image, 0); grid.SetColumn(image, 0);
+            var label = new Label { FontAttributes = FontAttributes.Bold, TextColor = Colors.White, Padding = 5, Text ="sdfg" , };
+            grid.SetRow(label, 0); grid.SetColumn(label, 1);
+            var boxView = new BoxView { BackgroundColor = Colors.Red, HeightRequest = 1, };
+            grid.SetRow(boxView, 1); grid.SetColumnSpan(boxView, 2);
+            var areaLabel = new Label { FontAttributes = FontAttributes.Bold, Text = "10,180,000 sq. km.", TextColor = Colors.White, };
+            grid.SetRow(areaLabel, 2); grid.SetColumnSpan(areaLabel, 2);
+            grid.Children.Add(image);
+            grid.Children.Add(label);
+            grid.Children.Add(boxView);
+            grid.Children.Add(areaLabel);
+            return new ViewCell { View = grid };
+        });
+    }
+
+public class Model
+{
+    public string Continent { get; set; }
+    public string Area { get; set; }
+    public int Population1 { get; set; }
+    public Model(string continent, int population, string area)
+    {
+        this.Continent = continent;
+        this.Population1 = population;
+        this.Area = area;
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Maps tooltip appearance customization](images/tooltip/tooltip_textStyle.png)
+
+N>
+* The tooltip displays the binding context value for shapes and bubbles.
+* Marker tooltip displays a Latitude and Longitude value.
```

Comment:
The BindingContext of the Shape and Bubble will be the corresponding underline object.
The BindingContext of the Marker will be the corresponding MapMarker.